pootle/core/decorators.py

```diff
@@ -132,15 +131,7 @@ def set_resource(request, path_obj, dir_path, filename):
 
     is_404 = False
 
-    vfolder = None
     clean_pootle_path = pootle_path
-    if 'virtualfolder' in settings.INSTALLED_APPS:
-        from virtualfolder.helpers import extract_vfolder_from_path
-        # Get a clean pootle path for retrieving the directory or store.
-        # A clean pootle path is a pootle path without any virtual folder name on
-        # it. For example /af/test_vfolders/browser/chrome/ is the corresponding
-        # clean pootle path for /af/test_vfolders/browser/vfolder8/chrome/
-        vfolder, clean_pootle_path = extract_vfolder_from_path(pootle_path)
```

Comment:
Given that this is dropped you can probably simplify the surrounding code a bit more now that `clean_pootle_path` and `pootle_path` are the same.